src/web/mjs/engine/Manga.mjs

```diff
@@ -210,4 +211,30 @@ export default class Manga extends EventTarget {
         count = Math.min( digits, count );
         return '0'.repeat( digits - count ) + text;
     }
+
+    /**
+     * Check newer content within bookmarks
+     */
+    async checkNewContent()
+    {
+        await this._getUpdatedChaptersFromWebsite();
```

Comment:
Every time a chapter is marked or unmarked,the chapter list is loaded from the website?
Not sure if this is a good idea, especially for websites that are loading the chapters very slow (e.g. 9anime, VRV)